### PR TITLE
Menu: Speed up pillar animation

### PR DESCRIPTION
**Pull request by gu-stav**
## What does this change?

`will-change` as compositing layer hint and a faster transition to feel less laggy.

## What is the value of this and can you measure success?

UX.

## Does this affect other platforms - Amp, Apps, etc?

No.

## Screenshots

No.

## Tested in CODE?

No.

### CODE REVIEW

File: static/src/stylesheets/layout/new-header/_pillars.scss [hunk 1]
```diff
@@ -37,11 +37,11 @@
     @include mq(desktop) {
         min-width: 0;
         padding-right: 16px;
-        transition: min-width 400ms ease;
+        transition: min-width 200ms ease;
+        will-change: min-width;
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_pillars.scss]: Perfect! 

Can I see a gif?

**Review by NataliaLKB — APPROVED** [review]
💯

**Comment by sndrs** [comment]
could this use transform instead of `min-width`? it's a much cheaper animation to perform (no layout to recalc).

seems like should be doable with a sass loop and `:nth-child` selector?

**Comment by paperboyo** [comment]
[This here](https://aerotwist.com/blog/flip-your-animations/) may or may not be helpful, I wouldn’t know. All I know is that it says “60fps” and I like that.

**Pull request merged**